configurations/drich_only.yml

```
@@ -1,4 +1,3 @@
 features:
-  # beampipe:
   pid:
-    drich:
+    drich
```

Sorry to break the pattern here, but the dRICH actually needs the colon. This is because it's used a bit differently in the `jinja` template to allow loading of customized dRICH compact files on the fly (for optimizers)
```suggestion
    drich:
```

---

Yeah, everything should need the colon.